Automattic/jetpack
# Search: Use wp-admin page to customize settings

#### Changes proposed in this Pull Request:
* Proof of concept. **Do not merge**!
* Adds a "Search Configuration" wp-admin page within the "Appearance" sidebar navigation group. This page can read/update Jetpack Search options. 
  <img width="1792" alt="Screen Shot 2021-05-18 at 3 37 32 PM" src="https://user-images.githubusercontent.com/4044428/118727698-66d1ed80-b7f0-11eb-99d2-e963d64da9ae.png">


#### Jetpack product discussion
pcNPJE-7J-p2.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
* Apply this change to your site with a Jetpack Search subscription.
* Build the assets via `yarn build`.
* Ensure that menu item `Search` (in `Jetpack`) has been added to the wp-admin sidebar.
* Ensure that the `Jetpack Search` link opens a wp-admin page for configuring Jetpack Search.
* Ensure that the configuration panel for the `Jetpack Search` page works as expected.
